templates/cluster_config_ami_builder.json.erb

```diff
@@ -152,7 +155,8 @@
         "instances": {
           "number_of_instances": 0,
           "instance_type": "<%= admin_instance_type %>",
-          "root_device_type": "ebs"
+          "root_device_type": "ebs",
+          "root_device_size": 16
```

Comment:
since these are all "16", does it make sense to define the default size somewhere more central?